libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1872,8 +1872,18 @@ void AP_OSD_Screen::draw_climbeff(uint8_t x, uint8_t y)
 void AP_OSD_Screen::draw_btemp(uint8_t x, uint8_t y)
 {
     AP_Baro &barometer = AP::baro();
-    float btmp = barometer.get_temperature(1);
-    backend->write(x, y, false, "%3d%c", (int)u_scale(TEMPERATURE, btmp), u_icon(TEMPERATURE));
+    float btmp = -300.0f;
```

Comment:
we could save a bit more flash by just not allowing for OSDn_BTEMP_EN when BARO_MAX_INSTANCES < 2. That can be done with an #ifdef in AP_OSD_Screen.cpp around the parameter declarations and around the btemp variable. The reason people set BARO_MAX_INSTANCES lower is to save flash space, so may as well save a bit more

Reply:
I've just changed to add #if.
Thanks!